lib/sufia/import/permission_builder.rb

```diff
@@ -3,20 +3,16 @@ module Sufia::Import
   class PermissionBuilder
     attr_reader :object
 
-    # @param PCDMObject object   Object for the permissions to be applied to
-    #
-    def initialize(object)
-      @object = object
-    end
-
     # Build permissions on a FileSet or a work based on the metadata from GenericFile
     #
-    #  @param Array[hash] generic_file_perms An array of hashes with the below keys
-    #  @option :agent    - Agent that has permisisons; example: "http://projecthydra.org/ns/auth/person#user1@example.com"
-    #  @option :mode     - access acl; example: "http://www.w3.org/ns/auth/acl#Write"
-    #  @option :acces_to - not used - Permissions are added to the object passed in the initializer
-    #  @option :id       - not used - Id for the permissions is generated
-    def build(generic_file_perms)
+    # @param PCDMObject object   Object for the permissions to be applied to
+    # @param Array[hash] generic_file_perms An array of hashes with the below keys
+    #   @option :agent    - Agent that has permisisons; example: "http://projecthydra.org/ns/auth/person#user1@example.com"
+    #   @option :mode     - access acl; example: "http://www.w3.org/ns/auth/acl#Write"
+    #   @option :acces_to - not used - Permissions are added to the object passed in the initializer
+    #   @option :id       - not used - Id for the permissions is generated
+    def build(object, generic_file_perms)
+      @object = object
```

Comment:
@hackmastera  know this is the smallest change, but I think it would be more obvious to just pass the object through to create on line 21 instead of assigning it to an instance variable.